.gitlab-ci.yml

```
@@ -376,7 +376,7 @@ test:acceptance:script:
   stage: publish
   image: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/debian:12
   tags:
-    - hetzner-amd-beefy
+    - mender-qa-worker-generic-light
```

If this is from a protected branch, you can use the hetzner protected runner instead:
```suggestion
    - hetzner-amd-beefy-privileged
```